projects/Buttons_and_ADC/src/main.c

```diff
@@ -0,0 +1,64 @@
+#include "adc.h"
+#include "delay.h"
```

Comment:
I don't think you need to use the delay library here? Should probably remove this